backendRoot/src/test/java/com/virtuslab/gitmachete/backend/integration/GitMacheteStatusTest.java

```diff
@@ -90,13 +89,13 @@ public void statusTestWithMultiRemotes() throws Exception {
   }
 
   private void createDirStructure() throws IOException {
-    Files.createDirectory(tmpTestDir);
-    Files.createDirectory(scriptsDir);
+    Files.createDirectories(scriptsDir);
   }
 
   private void copyScriptFromResources(String scriptName) throws URISyntaxException, IOException {
     URL resourceUrl = getClass().getResource("/" + scriptName);
     assert resourceUrl != null : "Can't get resource";
+    Files.deleteIfExists(repositoryBuildingScript);
     Files.copy(Paths.get(resourceUrl.toURI()), repositoryBuildingScript);
```

Comment:
eventually `REPLACE_EXISTING` can be added as 3rd parameter instead of line above.

Reply:
Done

Reply:
PS `eventually` means `ultimately, finally` and not `possibly` in English (that false friend applies btw not only to Polish but to many other European languages, see https://english.stackexchange.com/a/525692)

Reply:
Oh, yeah, my fault. Thanks  :)